app/src/main/java/com/wizeline/mobilenews/data/apiservice/NewscatcherApiService.kt

```diff
@@ -14,14 +16,12 @@ interface NewscatcherApiService {
      **/
     @GET(SEARCH_NEWS)
     suspend fun searchNews(
+        @Query("lang")
+        lang: String = LANGUAGE,
+        @Query("topic")
+        topic: String = TOPIC,
```

Comment:
This change has created a compilation error on the` NewsSearchPagingSource.kt `file. Because of the two new fields introduced, you may want to specify the named argument to prevent the error on line 21 of such file and test the solution.

Reply:
does it give you an error? the branch is working fine for me

Reply:
It gives me the same error. I think the fields "lang" and "topic" are missing in `retrofitService.searchNews(search, PAGE_SIZE, position)` query

Reply:
I see you posted a new commit, I'll check it again.

Reply:
It's the same error for me.

Reply:
is working fine now, just hit the "Re-request review" icon on the reviewers list and I'll approve it.

Reply:
It works for me now 💯